examples/github+tf_oss/s3_backend.hcl

```diff
@@ -0,0 +1,3 @@
+key    = "github-tf-oss-state/terraform.tfstate"
+bucket = "devoteam-mgmt-aft-terraform-state"
+region = "eu-west-1"
```

Comment:
I don't think it's necessary to put this in a separate file. You can just wrap it in the braces inside `backend "s3" {}`. Then we can just run `terraform init` without having to specify a config